### PR TITLE
test: add  query performance benchmark

### PR DESCRIPTION
**Pull request by gaoboal**
Added a set of benchmarking tests to compare the query performance of OpenMLDB and Redis. The tests use the open-source TalkingData dataset and cover the following typical scenarios:

1. Querying all data for a specified key.
2. Querying data at a specified time range for a given key.
3. Querying data at a specified timestamp for a given key.

### CODE REVIEW

File: benchmark/src/main/java/com/_4paradigm/openmldb/memoryusagecompare/BenchmarkQueryPerf.java [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+        String sql = "CREATE TABLE IF NOT EXISTS `" + tableName + "`(\n" +
+                "`ip` string, \n" +
+                "`app` int, \n" +
+                "`device` int, \n" +
+                "`os` int, \n" +
+                "`channel` int, \n" +
+                "`click_time` Timestamp, \n" +
+                "`is_attributed` int \n" +
+                ")\n" + "OPTIONS (replicanum=1); ";
```
Comment (gaoboal) [inline benchmark/src/main/java/com/_4paradigm/openmldb/memoryusagecompare/BenchmarkQueryPerf.java]: add ts

File: benchmark/src/main/java/com/_4paradigm/openmldb/memoryusagecompare/OpenMLDBExecutor.java [hunk 2]
```diff
@@ -123,6 +119,67 @@ void insertTalkingData(HashMap<String, ArrayList<TalkingData>> keyValues) {
...
+            statement = executor.getStatement();
+            statement.execute(sql);
+            res = statement.getResultSet();
+
+            ResultSetMetaData metaData = res.getMetaData();
+            int columnCount = metaData.getColumnCount();
+            ArrayList<HashMap<String, Object>> rows = new ArrayList<>();
+
+            while (res.next()) {
+                HashMap<String, Object> row = new HashMap<>();
```
Comment (gaoboal) [inline benchmark/src/main/java/com/_4paradigm/openmldb/memoryusagecompare/OpenMLDBExecutor.java:135]: 增加开关

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Comment by vagetablechicken** [comment]
I'll approve it after code update and get result set size test

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**